font.css

```diff
@@ -0,0 +1,6 @@
+@font-face {
+    font-family: "pt_serifbold";
+    src: url("ptserif-bold-webfont.woff") format("woff");
+    font-weight: normal;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)